java/client/src/main/java/glide/api/commands/HyperLogLogBaseCommands.java

```diff
@@ -70,4 +70,26 @@ public interface HyperLogLogBaseCommands {
      * }</pre>
      */
     CompletableFuture<Long> pfcount(String[] keys);
+
+    /**
+     * Merges multiple HyperLogLog values into a unique value.
+     *
+     * <p>The computed merged HyperLogLog is created if it does not exist before (defaulting to an
+     * empty HyperLogLog).<br>
+     * If the destination variable exists, it is treated as one of the source HyperLogLog data set.
+     *
+     * @see <a href="https://redis.io/commands/pfmerge/">redis.io</a> for details.
+     * @param destKey The destination where to merge HyperLogLog data sets into.
+     * @param sourceKeys The source HyperLogLog to merge.
```

Comment:
```suggestion
     * @param sourceKeys Identifiers of the HyperLogLog structures to be merged.
```

Reply:
I saw this comment was resolved but it looks like this line hasn't been changed? I agree with San that this line should indicate multiple HyperLogLog identifiers can be provided

Reply:
Oops, should be fixed now